tests/ensure_provider_tests.py

```diff
@@ -88,6 +88,12 @@
         EXTRA_RUN_CONDITION: "ansible_distribution_major_version == '7'",
     },
     "playbooks/tests_wireless_plugin_installation.yml": {},
+    "playbooks/tests_wireless_wpa3_owe.yml": {
+        "comment": "# OWE has not been exposed via D-Bus on RHEL 7(dist-tag) and \
+Fedora 32. Failed in setting up mock wifi on RHEL 8",
+        EXTRA_RUN_CONDITION: "ansible_distribution_major_version != '7' and \
+ansible_distribution_major_version != '32' and ansible_distribution != 'RedHat'",
+    },
```

Comment:
There are so many negations here, can you change it to allow only the supported distributions?

Reply:
Changed to only allow the supported distributions